### PR TITLE
Test weighted selection of environments

### PR DESCRIPTION
**Pull request by forabi**

### CODE REVIEW

File: src/assignEnvironment.test.ts [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+    const results = await testManyTimes(numTests);
+    const cookies = mapValues(
+      countBy(results.map(result => result.parsedCookies![0]!.env)),
+      v => v / numTests,
+    );
 
-  beforeEach(async () => {
-    context = await createTestContext();
+    expect(cookies.beta).toBeCloseTo(0.25, 1);
+    expect(cookies.master).toBeCloseTo(0.75, 1);
```
Comment (wholesomedev) [inline src/assignEnvironment.test.ts:16]: 😄 you just had to make sure, didn't you!

File: src/testHelpers.ts [hunk 2]
```diff
@@ -13,17 +13,17 @@ export type CreateTestContextOptions = {
   contextOverrides?: DeepPartial<Context>;
 };
 
-export type TestContext = Readonly<{
+export type TestResult = Readonly<{
```
Comment (wholesomedev) [inline src/testHelpers.ts:16]: I think this is a better name 👍

**Review by wholesomedev — COMMENTED** [review]
LGTM

File: src/assignEnvironment.test.ts (hunk 1 above)
Comment (forabi) [reply src/assignEnvironment.test.ts:16]: Yeah! It's easy to mess up. Tests make iteration way faster and safer.

Comment (forabi) [reply src/assignEnvironment.test.ts:16]: Ah! I realized you are referring to the exact numbers. It's just nice that we can prove this :smile:

**Pull request merged**